src/OpenColorIO/CPUInfo.cpp

```diff
@@ -182,6 +182,46 @@ CPUInfo& CPUInfo::instance()
     return singleton;
 }
 
+#elif defined(OCIO_APPLE_M1_ARCH_X86) || OCIO_USE_SSE2NEON
+CPUInfo::CPUInfo()
+{
+    flags = 0;
+    memset(name, 0, sizeof(name));
+
+    // Hardcode name to Apple ARM.
+    snprintf(name, sizeof(name), "%s", "Apple ARM");
+
+    // Note that Rosetta does not support any AVX instructions.
+    // See https://developer.apple.com/documentation/apple-silicon/about-the-rosetta-translation-environment#What-Cant-Be-Translated
+
+    // Also note that during testing on a Apple M1 MacBook, SSE 4.2 does not seems to be supported.
+
+#if !defined(__aarch64__)
+    // Enable SSE2 instructions support using Rosetta for the x86_64 architecture on Apple ARM cpu.
+    if (OCIO_USE_SSE2)
+    {
+        flags |= X86_CPU_FLAG_SSE2;
+    }
+    //TODO: Once the other SSE instructions are implemented into OCIO, these can be enabled here.
```

Comment:
why do we need to special case x86 for rosetta? Rosetta supports the `cpuid` instruction, can't we just use the regular path?

My M2 mac reports these extensions 
x86_64 VirtualApple @ 2.50GHz +sse2+sse3+ssse3+sse4+sse42

Reply:
We don't. I came to the same conclusion after looking at the failed CIs.

The next commit should fix the CI and Rosetta will use the regular path for x86.